planner/core/common_plans.go

```diff
@@ -591,6 +600,115 @@ REBUILD:
 	return err
 }
 
+var (
+	heapDumpInterval = time.Minute * 3
+	heapDumpTime     = time.Now()
+	debugThreshold   = 0.6 // 60%
```

Comment:
Can we decrease the threshold to make plan-cache-patch-log easier to trigger?

Reply:
decrease  to 40%, PTAL.